pkgs/tools/security/kwalletcli/default.nix

```diff
@@ -1,5 +1,5 @@
 { mkDerivation, fetchFromGitHub, lib, makeWrapper, pkgconfig
-, kcoreaddons, ki18n, kwallet, mksh, pinentry_qt5 }:
+, kcoreaddons, ki18n, kwallet, mksh, pinentry-qt5 }:
```

Comment:
```suggestion
, kcoreaddons, ki18n, kwallet, mksh, pinentry-qt }:
```

Reply:
Fixed. TBH, I don't understand why `kwalletcli` was pointed that hard to a specific pinentry flavor in 899e95582eefcfdaef156723f825a94c267de5f2, @peterhoeg can you elaborate?

Reply:
ping @peterhoeg